emsdk.py

```diff
@@ -256,7 +256,9 @@ def is_exe(fpath):
 
 def vswhere(version):
   try:
-    program_files = os.environ['ProgramFiles(x86)'] if 'ProgramFiles(x86)' in os.environ else os.environ['ProgramFiles']
+    program_files = os.environ.get('ProgramFiles(x86)')
```

Comment:
should we be consistent between `os.environ.get` and `os.getenv`? AFAICS they are the same, except that `os.environ` doesn't capture changes made to the environment (other than via `os.environ` itself) after program startup. I guess that's a bigger change than this PR though, and this code depends on `os.environ` throwing if neither key is in the environment at all...

Reply:
Yes exactly, I want this code to throw on lin 261.